contrib/apiage.py

```diff
@@ -311,6 +383,14 @@ def main():
             sys.exit(1)
         write_json(cli.current, api_tracked)
         write_markdown(cli.document, api_tracked)
+    elif cli.mode == "fix-versions":
+        values = {}
+        _setif(values, "added_in_version", cli.added_in_version)
+        _setif(values, "expected_stable_version", cli.stable_in_version)
+        _setif(values, "deprecated_in_version", cli.deprecated_in_version)
+        _setif(values, "expected_remove_version", cli.remove_in_version)
+        api_fix_versions(api_tracked, values=values, pred=_make_fix_filter(cli))
+        write_json(cli.current, api_tracked)
```

Comment:
We can update markdown also afterwards, right? Then we could avoid running the tool again just for generating markdown.

Reply:
Heh. We could, and in fact the `--mode=update` does, but I was feeling bad about that. Because looking at the code more recently I felt like it violated the "do one job" part of the old unix mantra of "do one job and do it well".  Especially, since the script is almost always invoked through the Makefile, I didn't want to overload the behavior of "fix-versions".